submissions/onyemeMichael/database/init.sql

```diff
@@ -73,6 +73,21 @@ INSERT INTO customers (firstname, lastname, email, customer_password)
 VALUES ('Paul', 'Onyeme', 'paulonyeme@email.com', 'paul'),
 	   ('Mayowa', 'Ojo', 'mayawaojo@email.com', 'mayowa');
 
+-- customers_datails table
+CREATE TABLE customer_details(
+        id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+        full_name VARCHAR(30) NOT NULL,
```

Comment:
Don't change the task. The task says `first_name` and `last_name`.  Don't change it to suit what you need